js/src/ui/AccountCard/accountCard.js

```diff
@@ -32,19 +34,22 @@ export default class AccountCard extends Component {
     className: PropTypes.string,
     disableAddressClick: PropTypes.bool,
     onClick: PropTypes.func,
-    onFocus: PropTypes.func
+    onFocus: PropTypes.func,
+    password: PropTypes.bool,
```

Comment:
`showPassword`? (On first read-through it looked like passing the actual password)